Resources/_template_week2_Haikus.md

```diff
@@ -0,0 +1,31 @@
+# Template for the haiku assignment
+
+## Instructions
+
+1. Form a subteam of three students from your team.
+2. As a subteam, create one branch on GitLab named `Group<groupnumber>/Haiku_<teamletter><subteamnumber>` (e.g. `Group1/Haiku_A1`).
+3. Copy this Template and rename it to the same name as the branch; keep the extension `.md` (e.g. `Group1/Haiku_A1.md`).
+4. One of the teammembers writes the first line of the haiku in the file on their local computer. Then uploads it to the branch.
+5. The next teammember pulls and edits the file, writing the second line of the haiku in the file. Then uploads it to the branch.
+6. The last teammember pulls and edits the file, and writing the third line of the haiku in the file. Then uploads it to the branch.
+7. One of the students creates a Merge Request to merge the file with the haiku into the master branch.
+8. The subteam assigns the other students from the team (but not from your subteam) to review the Merge Request.
+9. Once the reviewers have confirmed that it is a valid haiku, they can approve the Merge Request.
+10. One of the students from the subteam who wrote the haiku can merge the Merge Request.
```

Comment:
students have developer role, and not maintainer on gitlab. They cannot approve merge request on the main repo. This option is reserved for the maintainers: the instructors team